From cb19a3ca4848e62a55148d37ab79bb9ba2aada6d Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Wed, 29 Apr 2015 10:45:47 +0800
Subject: [PATCH 1005/1594] MLK-10767-3 ARM: dts: imx6ul: config lcdif clocks
 in dts file

commit 092042709d9f7041a121d64b70774257020e449b from
git://git.freescale.com/imx/linux-2.6-imx.git

The lcdif clocks are required to properly config after
the commit 14aa22162d9070caf44923511cd64b7f6e0b1f1d
is been pushed.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 2052543..575c336 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -828,8 +828,8 @@
 				compatible = "fsl,imx6ul-lcdif", "fsl,imx28-lcdif";
 				reg = <0x021c8000 0x4000>;
 				interrupts = <GIC_SPI 5 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX6UL_CLK_DUMMY>,
-					 <&clks IMX6UL_CLK_DUMMY>,
+				clocks = <&clks IMX6UL_CLK_LCDIF_PIX>,
+					 <&clks IMX6UL_CLK_LCDIF_APB>,
 					 <&clks IMX6UL_CLK_DUMMY>;
 				clock-names = "pix", "axi", "disp_axi";
 				power-domains = <&gpc 2>;
-- 
1.7.5.4

